examples/text_labels_and_annotations/custom_legends.py

```diff
@@ -0,0 +1,56 @@
+"""
+========================
+Composing Custom Legends
+========================
+
+Composing custom legends piece-by-piece.
+
+Sometimes you don't want a legend that is explicitly tied to data that
+you have plotted. For example, say you have plotted 10 lines, but don't
+want a legend item to show up for each one. If you simply plot the lines
+and call ``ax.legend()``, you will get the following:
+"""
+# sphinx_gallery_thumbnail_number 2
+from matplotlib import rcParams, cycler
+import matplotlib.pyplot as plt
+import numpy as np
+
+N = 10
+data = [np.logspace(0, 1, 100) + np.random.randn(100) + ii for ii in range(N)]
+data = np.array(data).T
+cmap = plt.cm.coolwarm
+rcParams['axes.prop_cycle'] = cycler(color=cmap(np.linspace(0, 1, N)))
+
+fig, ax = plt.subplots()
+lines = ax.plot(data)
+ax.legend(lines, loc=(1.05, .05), )
```

Comment:
I may be wrong but I think that one tries to use the leading zeros wherever possible: `(1.05, .05)` <- `(1.05, 0.05)`?